mern-ecommerce/controllers/auth.js

```diff
@@ -62,20 +62,20 @@ exports.requireSignin = expressJwt({
   userProperty: 'auth',
 });
 
-exports.isAuth = (req, res, next) => {
-  let user = req.profile && req.auth && req.profile._id == req.auth._id;
-  if (!user) {
+exports.isAdmin = (req, res, next) => {
+  if (req.profile.role === 0) {
     return res.status(403).json({
-      error: 'Access denied',
+      error: 'Admin resource! Access denied',
```

Comment:
Error message consistency: The error message 'Admin resource! Access denied' is not consistent with the error messages in the rest of the code. It would be better to use a similar format and structure for all error messages throughout the code.